### DISCUSSION TITLE
[mini.pair] How to restrict triggering pairs to only when pressing enter

### DISCUSSION THREAD

**Discussion by mcchrish** [nvim-mini/mini.nvim#952] [Q&A] [answered]
Similar to https://github.com/rstacruz/vim-closer

**Comment by echasnovski** [comment, marked as answer]
Unfortunately, there is no way of doing that with 'mini.pairs' and it is not planned. The whole pair is inserted immediately after pressing its opening character. Pressing enter creates an extra indented line if it is pressed exactly inside a pair.
